lib/Mojolicious/Command/Author/generate/callbackery_app/lib/App/GuiPlugin/Song.pm

```diff
@@ -25,6 +25,32 @@ All the methods of L<CallBackery::GuiPlugin::AbstractTable> plus:
 
 =cut
 
+
+has screenOpts => sub {
+    my $self = shift;
+    my $opts = $self->SUPER::screenOpts;
+    return {
+        %$opts,
+        # an alternate layout for this screen
+        layout => {
+            class => 'qx.ui.layout.Dock',
+            set => {},
+        },
+        # and settings accordingly
+        container => {
+            set => {
+                maxWidth => 700,
+                maxHeight => 500,
+                alignX => 'left',
+                alignY => 'top',
```

Comment:
Probably we could add some more properties like 
                marginLeft => 100,
                marginTop => 20,
Probably just as comment

Reply:
habe einen pointer auf die qooxdoo api dok eingebaut.